model/prisma/generator.js

```diff
@@ -0,0 +1,27 @@
+const helper = require('@prisma/generator-helper');
+const fs = require('fs');
+const path = require('path');
+
+helper.generatorHandler({
+  onManifest() {
+    return {
+      defaultOutput: '../node_modules/@prisma/client/runtime/index.js',
+      prettyName: 'Prisma Runtime Path Fixer',
+    };
+  },
+  async onGenerate(options) {
+    const output = options.generator.output?.value;
+
+    if (output) {
+      try {
+        const content = fs.readFileSync(path.resolve(output)).toString().replace(/util\/types/g, 'util/support/types');
```

Comment:
I don't know why I have to do it.

Reply:
`util/support/types` is [here](https://github.com/browserify/node-util/blob/master/support/types.js).

Where does original "util/types" refer...?

Reply:
> Where does original "util/types" refer...?

I found it in Node.js
https://nodejs.org/docs/latest-v16.x/api/util.html#utiltypes

Reply:
Specifying a fallback for resolve in the webpack config maybe also help.
ref. https://webpack.js.org/configuration/resolve/

Reply:
https://github.com/webpack/webpack/issues/11282